### DISCUSSION TITLE
Source Code to run this reports

### DISCUSSION THREAD

**Discussion by Binoxius** [RescueSocialTech/Amber-Heard_Disinformation_Operations_Bots#9] [General]
This report is not bad, but where is the proof of this all? Where is the source code to check the quality of this all? GitHub is a developer platform and not a report platform. Source code please.

Thank you

**Comment by christinataft** [comment]
Hi Binoxius, 
We include Jupiter notebooks and html files e.g., as well as some py files.
We do include scripts and code e.g., using the APIs and collecting data - https://github.com/RescueSocialTech/Amber-Heard_Disinformation_Operations_Bots/tree/main/Instagram%20Analysis/Instagram%20Scripts-Code
Reddit, Instagram, and Twitter is here - https://github.com/RescueSocialTech/Amber-Heard_Disinformation_Operations_Bots/tree/main/Analysis%20Cross-Platforms/Scripts%20-%20Twitter%2C%20Reddit%2C%20Instagram
There is Change.org code too from @sergia-ch and Facebook was basic from @MariyaMurova 

To create the reports, we also include all of the Jupiter notebooks - https://github.com/RescueSocialTech/Amber-Heard_Disinformation_Operations_Bots/tree/main/-%20Total%206%20Platforms%20Social%20Network%20Analysis and each folder...
It includes code to create the graphs.

I can include the basic scraper code pre-social network analysis, but it's not as high of quality. Can you point out more where source code is missing? I will request more code provided

**Reply by 868Dominus** [reply]
Hello Christiana,

I am sorry for my late reply, but for some reason it does not show up in my emails. First, it is a shame what the social media has done and has allowed here against AH. I am not a friend of AH or JD, but this blackmailing destroying not only the future of certain people, also innocent people as well, like the kiddo of AH. I think if you really want to prove something, then is it important to find any kind of online postings where only parts of the real thing was used. And this shows the real data. This can be done easy. 

There are many people, they think they are untouchable by law and even make money out of posting false or incomplete data. And this is what actually harms AH the most. And the tricky part is that it was made public. If I was you, I would take a closer look at what was allowed and from which platform. For better understanding. Most postings about China were not allowed, truth or not, and in case of AH it seems everything was allowed. So, if I was in your position, I would go on this matter in a complete different approach. And as far I see this reports here, there are not to see how many was out of the context. Make that sense?

AH had from the start on no change to win this "social media" war! But if this is played smart, it can turn the table.
